daemon/cmd/kube_proxy_replacement.go

```diff
@@ -346,6 +350,22 @@ func initKubeProxyReplacementOptions() (bool, error) {
 			}
 		}
 
+		if option.Config.EnableMKE {
```

Comment:
Shouldn't this be in the `if option.Config.EnableHostReachableServices {` branch above??